### PR TITLE
Adding Windows Networking updates

### PR DESCRIPTION
**Pull request by awsbpfeiff**
*Issue #, if available:*

*Description of changes:*
Updated Windows Networking content

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/windows/docs/ami.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Use Amazon EC2 Image Builder to select between Windows Server versions, AWS Wind
 
 **NOTE:** Prior to selecting a base image, consult the [Windows Server Version and License](licensing.md) section for important details pertaining to release channel updates.
 
+## Configuring faster launching for custom EKS optimized AMIs ##
+
+When using a custom EKS Optimized AMI, Windows worker nodes can be launched up to 65% faster by enabling the Fast Launch feature. This feature maintains a set of pre-provisioned snapshots which have the _Sysprep specialize_, _Windows Out of Box Experience (OOBE)_ steps and required reboots already completed. These snapshots are then used on subsequent launches, reducing the time to scale-out or replace nodes. **Fast Launch can be only enabled for AMIs you own** through the EC2 console or in the AWS CLI and the number of snapshots maintained is configurable. Note: **Fast Launch is not compatible with the default Amazon-provided EKS optimized AMI**, create a custom AMI as above before attempting to enable it.  For more information: [AWS Windows AMIs - Configure your AMI for faster launching](https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/windows-ami-version-history.html#win-ami-config-fast-launch)
```
Comment (jicowan) [inline content/windows/docs/ami.md:32]: Can you use an admonition for the note?  See https://python-markdown.github.io/extensions/admonition/. Basically you need to prepend the note with `!!! [note]`. You should be able to find examples of this in the guide.

**Review by jicowan — COMMENTED** [review]
Please use an admonition for the note, otherwise this looks good.

**Comment by andrewcr7** [comment]
I updated the Fast Launch section with an admonition styled similarly to another found in the guide.

File: content/windows/docs/ami.md [hunk 2]
```diff
@@ -27,6 +27,13 @@ Use Amazon EC2 Image Builder to select between Windows Server versions, AWS Wind
 
 **NOTE:** Prior to selecting a base image, consult the [Windows Server Version and License](licensing.md) section for important details pertaining to release channel updates.
 
+## Configuring faster launching for custom EKS optimized AMIs ##
+
+When using a custom EKS Optimized AMI, Windows worker nodes can be launched up to 65% faster by enabling the Fast Launch feature. This feature maintains a set of pre-provisioned snapshots which have the _Sysprep specialize_, _Windows Out of Box Experience (OOBE)_ steps and required reboots already completed. These snapshots are then used on subsequent launches, reducing the time to scale-out or replace nodes. Fast Launch can be only enabled for AMIs *you own* through the EC2 console or in the AWS CLI and the number of snapshots maintained is configurable. 
+> :warning: **Fast Launch is not compatible with the default Amazon-provided EKS optimized AMI**, create a custom AMI as above before attempting to enable it.
```
Comment (jicowan) [inline content/windows/docs/ami.md:33]: To write an admonition you use `!!! warning` You can see some examples [here](https://python-markdown.github.io/extensions/admonition/)

**Pull request merged**